app/models/product.rb

```diff
@@ -10,7 +14,12 @@ class Product < ApplicationRecord
   validates :title, :description, :image_url, presence: true
 
   # Adding validation related to price being positive number
-  validates :price, numericality: { greater_than_or_equal_to: 0.01 }
+  validates :price, 
+    numericality: { 
+      greater_than: :discount_price,
+      message: "must be greater than Discount price"
```

Comment:
use "must be greater than discount price"

Reply:
Use basic validations as well even if it's not mentioned in the exercise
ie- price should be greater than 0
Check for all attributes like discount price